examples/cloud-operations/network-dashboard/tests/test.tf

```diff
@@ -0,0 +1,293 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_folder" "test-net-dash" {
+  display_name = "test-net-dash"
+  parent       = "organizations/${var.organization_id}"
+}
+
+##### Creating host projects, VPCs, service projects #####
+
+module "project-hub" {
+  source          = "github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/project?ref=v14.0.0"
```

Comment:
I know I said before you should ping the module version but I thought we were in a different repo. You should really use relative paths here (i.e. "../../../../modules/project"). Sorry...